deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,217 @@
+import argparse
+import json
+import pandas as pd
+
+from datetime import datetime
+from os import makedirs
+from os.path import exists
+from subprocess import run
+
+# Global Configuration Variables
+csv_types = ['aggregate', 'intermediate']
+url_format = 'https://'
+
+def format_directory(directory):
+    """Check the paths of the directories and format them if necessary.
+    Args:
+        args (string): Path of a directory.
+    Returns:
+        string: Formatted path.
+    """
+    return directory if directory.endswith("/") else f'{directory}/'
+
+def check_csv_types(types):
+    """Check that the CSV type is valid.
+    Args:
+        types (list): List of types.
+    """
+    for value in types:
+        if (value not in csv_types):
+            raise Exception("CSV type is not valid") 
+        
+def create_directories(directory):
+    """Create directory that does not exist.
+    Args:
+        directory (str): Path of directory.
+    """
+    if not exists(directory):
+        makedirs(directory)
+
+def create_session_file(path, user):
+    """Create file to save the browser session.
+    Args:
+        path (str): Path of directory.
+        user (str): Username.
+    """
```

Comment:
Consider improving docstring readability. Please apply similar improvement to all the docstrings 
```suggestion
   """Creates a file to save the browser session.

    Args:
        path (str): The directory path where the session file will be saved.
        user (str): The username associated with the session.

    Returns:
        None
    """
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/4fee113a6384ad0a9d5dd204ecf37708f0632be5